operators/pkg/util/backuputil.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package util
```

Comment:
this one should put under operators/multiculsterobservability/pkg/util since it's only required for mco operator

Reply:
Fixed in latest commit